src/parse.cpp

```diff
@@ -541,6 +541,12 @@ expression* parser::parse_async_expression_only(token async_token) {
         this->skip();
         continue;
       }
```

Comment:
Should I add the `// async await => {}  // Invalid` comment here?

Reply:
But shouldn't the comment say `async (await) => {}`?